src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -58,6 +68,32 @@
  *   &lt;property name="offset" value="2"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <pre>
+ * &#47;**
+ *  * @tag comment..
+ *  * Indentation spacing is 0 (violation)
+ *  *   Indentation spacing is 2 (OK)
+ *  *  Indentation spacing is 1 (violation)
```

Comment:
Add one more line with 4 spaces indent

Reply:
I don't know if its a problem or some misunderstanding on my part but this check does not show violations for identation  >= offset. Please confirm this. Also if it's not a problem, then I think it should be added to documentation as there is nothing mentioned about this.

Reply:
@strkkk also review this, please.

Reply:
What do you mean? Is it a case when indentation should be 2, but in fact it is 4 results in no violation?

If it is, you can check test inputs for cases like this. If there are some, you can see how it works. In case there are none, issue can be created.

Reply:
@strkkk yes, it does not show violations, Issue #2813 is about this problem.

Reply:
(sigh)
Ok, please ignore this item then, I was not aware of this issue